generate/downloader/download.go

```diff
@@ -72,9 +72,11 @@ var ds = []Downloader{
 	GrpcSwift{},
 	GrpcPython{},
 	//use npm ts-protoc-gen package
-	Ts{ID: "ts", ModuleName: "ts-protoc-gen", BinaryName : "protoc-gen-ts"},
+	Ts{ID: "ts", ModuleName: "ts-protoc-gen", BinaryName: "protoc-gen-ts"},
 	//use npm ts-proto package
-	Ts{ID: "ts_proto", ModuleName: "ts-proto", BinaryName : "protoc-gen-ts_proto"},
+	Ts{ID: "ts_proto", ModuleName: "ts-proto", BinaryName: "protoc-gen-ts_proto"},
+	//use npm protoc-gen-ts package
+	Ts{ID: "protoc-gen-ts", ModuleName: "protoc-gen-ts", BinaryName: "protoc-gen-ts"},
```

Comment:
Hi @zamronypj , just to make sure that ``BinaryName: "protoc-gen-ts"`` is different with line 75.

Reply:
@maitungmn  they are different binaries although they have same name.